Modules/EMCAL/src/RawErrorCheck.cxx

```diff
@@ -41,6 +37,16 @@ namespace o2::quality_control_modules::emcal
 void RawErrorCheck::configure()
 {
   mGeometry = o2::emcal::Geometry::GetInstanceFromRunNumber(300000);
+
+  // switch on/off messages on the infoLogger
```

Comment:
Were you aware that you can use custom severity values and select the severity cut off threshold in the QC config file?

Reply:
I am fine with it as it is though.

Reply:
No, I was not, because that affects also another checker which we want to mute on the infoLogger while we still tune the checkers. But for the moment can we go ahead with this and I might change it in a second iteration?

Reply:
sure